src/lib/webappSettings.ts

```diff
@@ -1,13 +1,25 @@
 import compiledCircuit from "../../axiom/data/compiled.json";
 import inputs from "../../axiom/data/inputs.json";
 import AverageBalanceAbi from "./abi/AverageBalance.json";
+import { chainIdToExplorerBaseUrl, getProviderClientSide } from "./utils";
+
+export const PROJECT_ID = process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID as string;
+export const CHAIN_ID = "11155111";
```

Comment:
Should this be dynamic too?

Reply:
we actually search for this string in `create-axiom-client` to replace it with the selected chain id value